### DISCUSSION TITLE
Trying (and failing) to intentionally "bleed" double with characters

### DISCUSSION THREAD

**Discussion by windowsyl** [kovidgoyal/kitty#5067] [Q&A] [answered]
I'm trying to use the \ue23a anatomical heart character in my prompt for reasons. I was using iterm before and (with the following space) the bleeding made it look normal but in kitty it's scaled down. I was looking through previous discussion and was basically very confused. The FAQ on the website said that it would be rendered normal-size if it's followed by a space, i tried having no narrow_symbols in my config and narrow_symbols U+1FAC0 2, neither of which helped. I remembered a previous discussion mentioning make character + spc a ligature but I have no clue about how to do that.
My actual prompt is more complicated but i can reproduce it with PROMPT=$'\ue23a ' in my zshrc.

I'm on mac using zsh and my font is Fira Code if it matters. I'm not making this an issue because I'm assuming this is user error, but I have no idea how to fix it. Thanks!

**Comment by kovidgoyal** [comment]
Works for me
![screenshot](https://user-images.githubusercontent.com/1308621/167285168-b346a759-d286-404b-b34d-426011494772.png)

**Reply by windowsyl** [reply]
Is there anything I could have set on accident that would turn it off? i have no narrow_symbols in my config, is there anything else you know of?
![image](https://user-images.githubusercontent.com/70595487/167308440-b7e91280-ff5c-4d57-b52a-b1324ad1bf7b.png)

**Comment by kovidgoyal** [comment, marked as answer]
No, if I had to guess your symbol is simply a tall one and so increasing
the number of cells wont cause it to become wider since kitty preserves
the aspect ratio when rendering symbols.
